application/gqlgen/meeting.go

```diff
@@ -92,11 +92,19 @@ func (r *meetingResolver) ImageFile(ctx context.Context, obj *models.Meeting) (*
 		return nil, nil
 	}
 
-	image, err := obj.ImageFile()
+	if !obj.FileID.Valid {
+		return nil, nil
+	}
+
+	image, err := dataloader.For(ctx).FilesByID.Load(obj.FileID.Int)
 	if err != nil {
 		return nil, domain.ReportError(ctx, err, "GetMeetingImage")
 	}
 
+	if err := image.RefreshURL(); err != nil {
+		return nil, err
+	}
+
```

Comment:
Did you weigh the alternative of doing this (RefreshURL) in the fetch method? At first glance, it seems that would be better. We might be able to do away with it altogether as it's only used in dev and test. So if we recognize and accept this limitation it could simplify production code.